### ISSUE TITLE
Add help link to 2fa-login view

### ISSUE THREAD

**Issue by kimberscott** [lookit/lookit-api#650]
# TL;DR
Replace the "New to Lookit? Register your family" link on /2fa-login/ with a link to troubleshooting info about 2fa (probably https://lookit.readthedocs.io/en/develop/researchers-log-in.html#troubleshooting but we may make a new section about 2fa troubleshooting)

# Narrative
As a researcher, if I'm running into trouble entering my OTP, I want to know what to do. (This has been a source of common confusion - people expect they should be able to reset it via password reset, etc.) The docs have info but not everyone finds it quickly.

# Acceptance Criteria
- [ ] Link to 2FA help is displayed on /2fa-login view in place of "register your family" link below:

![Screen Shot 2021-03-26 at 9 03 20 AM](https://user-images.githubusercontent.com/9554627/112635390-24a0b700-8e12-11eb-97bd-a855020319e4.png)

# Implementation Notes
Fine to hard-code the link for now. If we find we're adding a bunch of links to help pages, it might be sensible to look into how people do that well - e.g. name the link & add a urls table to the DB so we can easily edit these without redeploying.
